neo/Ledger/Blockchain.cs

```diff
@@ -328,7 +328,7 @@ private RelayResultReason OnNewConsensus(ConsensusPayload payload)
         {
             if (!payload.Verify(currentSnapshot)) return RelayResultReason.Invalid;
             system.Consensus?.Tell(payload);
-            RelayCache.Add(payload);
+            ConsensusRelayCache.Add(payload);
```

Comment:
Maybe we can move this call before system.Consensus?.Tell it could be useful for search consensus payloads in the future...

Reply:
I think so, @shargon. I do not see problems in caching it before processing, since it already passed `payload.Verify`

Maybe let's do this in another PR, right?